README.md

```diff
@@ -21,6 +21,7 @@ SDK to be used to access Pasqal Cloud Services.
 | 0.12.5  | 2024-11-12   | 2025-11-12     |
 | 0.12.6  | 2024-12-12   | 2025-12-12     |
 | 0.12.7  | 2025-01-14   | 2026-01-14     |
+| 0.20.2  | 2024-02-21   | 2026-02-21     |
```

Comment:
This timeline needs to be edited for pulser pasqal old versions that we still want to support